gradle/libs.versions.toml

```diff
@@ -8,6 +8,7 @@ composeBom = "2024.12.01"
 constraintlayoutCompose = "1.1.0"
 coreKtx = "1.15.0"
 coreSplashscreen = "1.0.1"
+datastorePreferencesAndroid = "1.1.1"
```

Comment:
We probably don't need the "Android" suffix here. BTW, androidx.datastore is multiplatform.